hunting/windows/docs/detect_dll_hijack_via_masquerading_as_microsoft_native_libraries.md

```diff
@@ -0,0 +1,62 @@
+# DLL Hijack via Masquerading as Microsoft Native Libraries
+
+---
+
+## Metadata
+
+- **Author:** Elastic
+- **Description:** This hunt identifies when a process loads a DLL normally located in `System32` or `SysWOW64` folders from an unusual path. Adversaries may execute their own malicious payloads by side-loading malicious DLLs. The hosts count also should help exclude false-positives by looking at low occurrences when this abnormal behavior is limited to unique agents.
+- **UUID:** `87c97865-fdaa-48b2-bfa6-67bed7cf56ef`
+- **Integration:** [endpoint](https://docs.elastic.co/integrations/endpoint)
```

Comment:
Need to add the other integration: I wonder if we should add a check for this.

Reply:
Fixed.